README.md

```diff
@@ -1,25 +1,330 @@
-# Cortex SDK
+# Cortex Click Node.js SDK
 
-An AI accelerated go-to-market (GTM) platform. Easily automate sales and marketing workflows from top of funnel content, to point of sale, support, and expansion.
+```
+npm install @cortexclick/cortex
+```
+
+The Cortex SDK let's you automate sales and marketing scenarios and accelerate your GTM motion with AI.
```

Comment:
probably best to still explain the acronym for GTM here.

Reply:
👍 https://github.com/cortexclick/cortex-sdk/pull/17